parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -502,6 +502,7 @@ impl pallet_collator_selection::Config for Runtime {
 }
 
 impl pallet_asset_tx_payment::Config for Runtime {
```

Comment:
`parachains/runtimes/testing/penpal/src/lib.rs` also needs this.

Reply:
Ok.